src/executor.cpp

```diff
@@ -1038,4 +1040,52 @@ void Executor::execute74()
     displayExecutionStatus(74, std::vector<types::FunctionNumber>(), true);
 }
 
-} // namespace panel
\ No newline at end of file
+types::DbusExecStatus
+    Executor::executeDbusRequests(const types::FunctionNumber funcNum)
+{
+    isThisReqFromDbus = true;
+    try
+    {
+        switch (funcNum)
+        {
+            case 21:
+            case 22:
+            case 34:
+            case 65:
+            case 66:
+            case 67:
+            case 68:
+            case 69:
+            case 70:
+                sendFuncNumToPhyp(funcNum);
+                break;
+
+            default:
+                std::runtime_error("Given function number is not supported to "
+                                   "execute via d-bus.");
+        }
+    }
+    catch (std::exception& e)
+    {
+        isThisReqFromDbus = false;
+        storeDbusExecStatus(false, "", "");
```

Comment:
Call the class level variable and assign values in-place

Reply:
done